packages/neon/neon/lib/src/models/account.dart

```diff
@@ -77,6 +77,18 @@ class Account implements Credentials {
     // Maybe also show path if it is not '/' ?
     return '$username@${uri.port != 443 ? '${uri.host}:${uri.port}' : uri.host}';
   }
+
+  Uri completeUri(final Uri uri) {
+    final result = Uri.parse(serverURL).replace(
+      queryParameters: uri.queryParameters,
+      fragment: uri.fragment,
+    );
+    return result.replace(
+      path: uri.hasAbsolutePath ? uri.path : '${result.path}/${uri.path}',
+    );
+  }
+
```

Comment:
I think UriBuilder from https://pub.dev/packages/uri could clean this up a bit.

Please also add doc comments as after 5 minutes I still don't understand what this is doing

Reply:
I was using that, but then found it unnecessary as it was possible with just Uri alone. Will add a doc comment.